plugins/inputs/gnmi/gnmi_test.go

```diff
@@ -12,11 +12,14 @@ import (
 	"testing"
 	"time"
 
+	jnprHeader "github.com/influxdata/telegraf/plugins/inputs/gnmi/extensions/jnpr_gnmi_extention"
```

Comment:
It's preferred to keep all telegraf internal packages to import together. Could you move it to the correct place?

Reply:
I'm sorry, didn't catch. Where do you want I move the go extension ?

Reply:
To the section with the others from `github.com/influxdata/telegraf`

Reply:
I've pushed the change